components/Navbar/UserMenu.tsx

```diff
@@ -11,6 +12,13 @@ export default function UserMenu({ setShowUserMenu }: UserMenuProps) {
     setShowUserMenu(false)
     router.push(path)
   }
+
+  const logout = async (e) => {
+    e.preventDefault()
+    localStorage.removeItem(TOKEN_KEY)
+    handleNavigation('/')
```

Comment:
This should also be calling a logout function in the API to blacklist this token